ginga/util/catalog.py

```diff
@@ -200,6 +179,7 @@ def search(self, **params):
 
 
 class AstroQueryImageServer(object):
```

Comment:
Ah, didn't realize it is also "AstroQuery" and not "Astroquery" here but maybe this is out of scope.

Reply:
We can provide some backward compatible classes that are just wrappers around the new class names and constructors, I think.